aio/content/examples/dependency-injection/src/app/test.component.ts

```diff
@@ -22,8 +22,8 @@ export class TestComponent {
 /////////////////////////////////////
 function runTests() {
 
-  const expectedHeroes = [{name: 'A'}, {name: 'B'}]
-  const mockService = <HeroService> {getHeroes: () => expectedHeroes }
+  const expectedHeroes = [{name: 'A'}, {name: 'B'}];
+  const mockService = <HeroService> {getHeroes: () => expectedHeroes };
```

Comment:
I think that if you do the following you don't have to disable the `@typescript-eslint/consistent-type-assertions` rule:

```suggestion
  const mockService = {getHeroes: () => expectedHeroes } as HeroService;
```

Reply:
yeah, you're totally right, sorry about this :sweat: 

(and thanks for noticing it! :slightly_smiling_face:)